### PR TITLE
Remove code

### LINKED ISSUE #206: Remove code from PortRegistry

### PR DESCRIPTION
**Pull request by hujw77**
- Fix #206

### CODE REVIEW

File: src/PortRegistry.sol [hunk 1]
```diff
@@ -26,41 +26,41 @@ import "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
...
     function initialize(address dao) public initializer {
         _transferOwnership(dao);
     }
 
     function _authorizeUpgrade(address) internal override onlyOwner {}
 
-    /// @dev Fetch port address by chainId and port code.
-    function get(uint256 chainId, bytes4 code) external view returns (address) {
-        return _portLookup[chainId][code];
+    /// @dev Fetch port address by chainId and name code.
```
Comment (jiguantong) [inline src/PortRegistry.sol:43]: name code => port name

File: src/PortRegistry.sol [hunk 2]
```diff
@@ -26,41 +26,41 @@ import "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
...
+    function get(uint256 chainId, string calldata name) external view returns (address) {
+        return _portLookup[chainId][name];
     }
 
     /// @dev Fetch port code by chainId and port address.
```
Comment (jiguantong) [inline src/PortRegistry.sol:48]: Fetch port name

File: src/PortRegistry.sol [hunk 3]
```diff
@@ -26,41 +26,41 @@ import "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
...
 
     /// @dev Set a port.
-    function set(uint256 chainId, bytes4 code, address port) external onlyOwner {
-        require(code != bytes4(0), "!code");
-        require(port != address(0), "!port");
-        _portLookup[chainId][code] = port;
-        _codeLookup[chainId][port] = code;
-        emit SetPort(chainId, code, port);
+    function set(uint256 chainId, string calldata name, address port) external onlyOwner {
+        _portLookup[chainId][name] = port;
```
Comment (jiguantong) [inline src/PortRegistry.sol:55]: Check port and name is not null?

File: src/interfaces/IPortRegistry.sol [hunk 4]
```diff
@@ -18,6 +18,6 @@
 pragma solidity ^0.8.0;
 
 interface IPortRegistry {
-    function get(uint256 chainId, address port) external view returns (bytes4);
-    function get(uint256 chainId, bytes4 code) external view returns (address);
+    function get(uint256 chainId, address port) external view returns (string memory);
+    function get(uint256 chainId, string calldata code) external view returns (address);
```
Comment (jiguantong) [inline src/interfaces/IPortRegistry.sol:22]: code => name

File: src/xAccount/XAccountFactory.sol [hunk 5]
```diff
@@ -84,31 +84,34 @@ contract XAccountFactory is Ownable2Step, Application, PortMetadata {
     }
 
     function isRegistred(address port) public view returns (bool) {
-        return REGISTRY.get(LOCAL_CHAINID(), port) != bytes4(0);
+        return bytes(REGISTRY.get(LOCAL_CHAINID(), port)).length > 0;
```
Comment (jiguantong) [inline src/xAccount/XAccountFactory.sol:87]: What's the difference between `bytes(REGISTRY.get(LOCAL_CHAINID(), port)).length > 0` and `REGISTRY.get(LOCAL_CHAINID(), port).length > 0`?

Comment (hujw77) [reply src/xAccount/XAccountFactory.sol:87]: `string` has not `.length` attribution.

**Review by jiguantong — APPROVED** [review]

**Pull request merged**